docs/reference/indices/index-templates.asciidoc

```diff
@@ -40,6 +40,7 @@ template with the highest priority is used.
 following index patterns:
 
 // tag::built-in-index-template-patterns[]
+- `.kibana-reporting`
```

Comment:
This one currently has a star at the end:
```suggestion
- `.kibana-reporting*`
```

Reply:
Thanks for the close look, @dakrone! I have committed your suggestion.